### PR TITLE
Readme and Changelog added

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]:
```suggestion
This file lists all the notable changes to this project.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Theme Designer 19.1.1 - 2019-03-07
+
+
+### New Features
+
+* Incremental build.
+
+* Output window now can display the  **Build Output**, **Preview Output**, and **Logs**.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]:
```suggestion
* The output window now can display the  **Build Output**, **Preview Output**, and **Logs**.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    <img src="media/Changelogs/19.1-output-window.png" width="50%" alt="Theme Designer 19.1 - Output Window"/>
+
+
+### Enhancements
+
+* Application layout has changed to the borderless.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]:
```suggestion
* The application's layout has changed to borderless.
```

Comment (dirk-pieterse) [reply CHANGELOG.md]: Why 'the' borderless?

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+* Theme Designer remembers opened files in a theme and opens these files after you reopen the theme.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]:
```suggestion
* The Theme Designer remembers opened files in a theme and opens these files after you reopen the theme.
```

Comment (dirk-pieterse) [reply CHANGELOG.md]: 'opened' 'opens' 'reopen'? Rewrite.

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+* Resolve Merge Conflicts - you can choose a file version without using the KDiff3 tool.
```
Comment (dirk-pieterse) [inline CHANGELOG.md]:
```suggestion
* Resolved Merge Conflicts - you do not need to use the KDiff3 tool to choose a file version.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+----
+## Table of Contents
+
+* [Overview](#overview)
+* [Features](#features)
+* [Getting Started](#getting-started)
+* [Get Help](#get-help)
+
+## Overview
+The DevExpress WPF Theme Designer was designed to reduce the grunt work associated with custom app-wide theme development. It is a free standalone tool that allows you to generate custom themes by altering the base colors of existing DevExpress WPF Themes.
```
Comment (dirk-pieterse) [inline README.md]: "the grunt work"??

File: CHANGELOG.md (hunk 4 above)
Comment (friedcucumber) [reply CHANGELOG.md]: @dirk-pieterse 
When you close a theme solution, Theme Designer remembers the files you had open previously and restores them when you re-open the theme solution.

File: README.md (hunk 6 above)
Comment (dirk-pieterse) [reply README.md]: Rewrite.

Comment (friedcucumber) [reply README.md]: Yep. I've copied this para from the website.
https://www.devexpress.com/products/net/controls/wpf/themes/

File: README.md [hunk 7]
```diff
@@ -1 +1,44 @@
...
+
+[![WPF Theme Designer YouTube video](media/youtube-video-thumbnail.png)](https://www.youtube.com/watch?v=c5HJj8UXp8g)
+
+## Features
+* [Theme Customization](https://docs.devexpress.com/WpfThemeDesigner/119320/workspace/tools) <br/> The Theme Designer offers a number of features designed to improve your editing experience. Pick colors from any on-screen application, filter element colors and use other new tools to easily colorize any visual element of your application.
+
+* [Palette](https://docs.devexpress.com/WpfThemeDesigner/400319/edit-colors) <br/> The latest DevExpress Themes contain a palette. 
+
+* [XAML Edit](https://docs.devexpress.com/WpfThemeDesigner/400298/edit-theme-in-xaml) <br/> The WPF Theme Designer empowers you to locate and modify control templates, styles and apply color presets. You can edit XAML files in the built-in Code View, or open the theme solution in Visual Studio.
```
Comment (dirk-pieterse) [inline README.md]: empowers you? Why are you writing this?

File: README.md [hunk 8]
```diff
@@ -1 +1,44 @@
...
+* [Getting Started](#getting-started)
+* [Get Help](#get-help)
+
+## Overview
+The DevExpress WPF Theme Designer was designed to reduce the grunt work associated with custom app-wide theme development. It is a free standalone tool that allows you to generate custom themes by altering the base colors of existing DevExpress WPF Themes.
+
+[![WPF Theme Designer YouTube video](media/youtube-video-thumbnail.png)](https://www.youtube.com/watch?v=c5HJj8UXp8g)
+
+## Features
+* [Theme Customization](https://docs.devexpress.com/WpfThemeDesigner/119320/workspace/tools) <br/> The Theme Designer offers a number of features designed to improve your editing experience. Pick colors from any on-screen application, filter element colors and use other new tools to easily colorize any visual element of your application.
```
Comment (dirk-pieterse) [inline README.md]: "designed to improve your editing experience."?

Comment (dirk-pieterse) [reply README.md]: and why 'easily'?

Comment (dirk-pieterse) [reply README.md]:
```suggestion
* [Theme Customization](https://docs.devexpress.com/WpfThemeDesigner/119320/workspace/tools) <br/> The Theme Designer allows you to pick colors from any on-screen application, filter element colors, and use other tools to paint your application's visual elements.
```

File: README.md [hunk 9]
```diff
@@ -1 +1,44 @@
...
+
+* [Palette](https://docs.devexpress.com/WpfThemeDesigner/400319/edit-colors) <br/> The latest DevExpress Themes contain a palette. 
+
+* [XAML Edit](https://docs.devexpress.com/WpfThemeDesigner/400298/edit-theme-in-xaml) <br/> The WPF Theme Designer empowers you to locate and modify control templates, styles and apply color presets. You can edit XAML files in the built-in Code View, or open the theme solution in Visual Studio.
+
+* Live Preview <br/> Flexibility is at the heart of our WPF Theme Designer. Once you've made your changes, you don't need to rebuild your entire project to view the result. A comprehensive set of previews and an ability to preview changes in control and demo views give you complete picture of your theme's appearance. Simply use the Designer's Preview window to evaluate the usability of your new Theme.
```
Comment (dirk-pieterse) [inline README.md]: "is at the heart of our WPF Theme Designer"? Are you writing a technical document, or a marketing brochure?

Comment (dirk-pieterse) [reply README.md]: Did you write this?

File: README.md [hunk 10]
```diff
@@ -1 +1,44 @@
...
+
+* [Import. Export. Share.](https://docs.devexpress.com/WpfThemeDesigner/118787/build-and-export-new-themes) <br/> The WPF Theme Designer ships with multiple options to save modified themes and retain them for later use. New themes can be built as assemblies or saved to a *.tdsln file. All changes made to individual color schemes remain consistent whenever you upgrade to a newer version of DevExpress WPF Controls.
+
+## Getting Started
+
+Download and install the WPF Theme Designer, then [get started](https://docs.devexpress.com/WpfThemeDesigner/118594/getting-started).
+
+## Get Help
+
+You can ask any questions in our [Support Center](https://www.devexpress.com/Support/Center) or [Online Chat](https://www.devexpress.com/).
```
Comment (dirk-pieterse) [inline README.md]: any questions? Do you mean 'any question'?

**Pull request merged**